### PR TITLE
Parts Localization support

### PR DESCRIPTION
**Pull request by tinygrox**
I made some localization changes to the parts

### CODE REVIEW

File: FOR_RELEASE/GameData/UmbraSpaceIndustries/Konstruction/Parts/PAL_Cradle250.cfg [hunk 1]
```diff
@@ -40,10 +40,10 @@ PART
 	MODULE
 	{
 		name = ModuleAnimateGeneric
-		animationName = CradleLeg0
-		startEventGUIName = Lower Leg 0
-		endEventGUIName = Raise Leg 0
-		actionGUIName = Toggle Leg 0
+		animationName = #LOC_USI_Animation_CradLeg0//CradleLeg0
```
Comment (Kerbas-ad-astra) [inline FOR_RELEASE/GameData/UmbraSpaceIndustries/Konstruction/Parts/PAL_Cradle250.cfg]: As above, these are internal variables and should not be localized.

File: FOR_RELEASE/GameData/UmbraSpaceIndustries/Akita/Parts/Akita_Cab.cfg [hunk 2]
```diff
@@ -50,7 +50,7 @@ PART
 		lightBrightenSpeed = 2.5
 		lightDimSpeed = 2.5
 		resourceAmount = 0.04
-		animationName = AkitaLights
+		animationName = #LOC_USI_Animation_AkitaLights//AkitaLights
```
Comment (Kerbas-ad-astra) [inline FOR_RELEASE/GameData/UmbraSpaceIndustries/Akita/Parts/Akita_Cab.cfg]: ```animationName``` is an internal variable.  It shouldn't be localized.
```suggestion
		animationName = AkitaLights
```

File: FOR_RELEASE/GameData/UmbraSpaceIndustries/Konstruction/Parts/PAL_Cradle250.cfg [hunk 3]
```diff
@@ -68,10 +68,10 @@ PART
 	MODULE
 	{
 		name = ModuleAnimateGeneric
-		animationName = CradleLeg2
-		startEventGUIName = Lower Leg 2
-		endEventGUIName = Raise Leg 2
-		actionGUIName = Toggle Leg 2
+		animationName = #LOC_USI_Animation_CradLeg2//CradleLeg2
```
Comment (Kerbas-ad-astra) [inline FOR_RELEASE/GameData/UmbraSpaceIndustries/Konstruction/Parts/PAL_Cradle250.cfg]:
```suggestion
		animationName = CradleLeg2
```

File: FOR_RELEASE/GameData/UmbraSpaceIndustries/Konstruction/Parts/PAL_Cradle250.cfg (hunk 1 above)
Comment (tinygrox) [reply FOR_RELEASE/GameData/UmbraSpaceIndustries/Konstruction/Parts/PAL_Cradle250.cfg]: Thank you for correction

**Comment by tinygrox** [comment]
That should be done

**Pull request merged**